CHANGELOG.md

```diff
@@ -1,7 +1,35 @@
 # Changelog
 
-All notable changes to this project will be documented in this file.
-This project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
+## vNext
```

Comment:
Is version updated later?

Reply:
It will be part of the release process. This PR is only an update of a dependency.